recipes/turbo_turtle/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "turbo_turtle" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lanl-aea/turbo-turtle/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e89f218c8d95484f3b0b8b61427c0bd82779226301436c991535966bac4d140d
+
+build:
+  noarch: python
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  entry_points:
+    - turbo-turtle = turbo_turtle._main:main
+  ignore_run_exports:
+    - python_abi
+
+requirements:
+  build:
+    - lazy_loader >=0.4
+    - scons >=4
+    - setuptools_scm[toml] >=6.2
+    - sphinx >=5.1
+    - sphinx-argparse
+    - sphinx-copybutton >=0.5.1
+    - sphinx_rtd_theme >=1
+    - sphinxcontrib-bibtex
+    - waves >=0.7.2,<0.11.0
+  host:
+    - python
+    - python-build
+    - pip
+    - setuptools >=62
+    - setuptools_scm[toml] >=6.2
+  run:
+    - python >=3.8
+    - lazy_loader >=0.4
+    - scons >=4
+    - waves >=0.7.2,<0.11.0
+
+test:
+  requires:
+    - pytest
+    - pytest-xdist
+  imports:
```

Comment:
Please also run pip check in the tests.

Reply:
This isn't a PyPI package. It's being built directly from the GitHub release source. Will ``pip check`` have any affect?

https://conda-forge.org/docs/maintainer/adding_pkgs/#pip-check

Reply:
Yes, but it is a Python package and `pip check` only checks for Python package metadata consistency and doesn't even contact PyPI for that.

Reply:
Got it. I've made the change and push soon. For my own understanding, ``pip check`` understands how to search and check packages installed in a conda environment, not just pip environments?

Reply:
It only understands pip environments, but conda environments with pip installed are also "pip environments" metadata-wise. This only checks the subset of dependency constraints that is exposed to pip. We use this to partially ensure that pip and conda constraints are inline.

Reply:
awesome. Thanks!